sdk/vault-bundles/src/action.ts

```diff
@@ -15,13 +16,22 @@ export interface ActionContext {
    * The reason specified in the request.
    */
   reason: string;
+  /**
+   * The HTTP request body of the invoke action API request serialized from JSON as a JavaScript object.
+   */
+  body: unknown;
+  /**
+   * A vault object that exposes methods to be used with Vault API from the action.
```

Comment:
"A vault object" means nothing to the reader. Maybe "An object that exposes..." is better